app/move/controllers/confirmation.js

```diff
@@ -0,0 +1,31 @@
+const { mountpath: movesUrl } = require('../../moves')
+const filters = require('../../../config/nunjucks/filters')
+
+function confirmation(req, res) {
+  const { move } = res.locals
+  const {
+    date,
+    person,
+    reference,
+    move_type: moveType,
+    to_location: toLocation,
+  } = move
+
+  const locals = {
+    moveReference: reference,
+    fullname: person.fullname,
+    location:
+      moveType === 'prison_recall'
+        ? req.t('fields::move_type.items.prison_recall.label')
+        : toLocation.title,
+    date: filters.formatDateWithDay(date),
+    action: {
+      createNewMoveUrl: '/move/new',
+      dashboardUrl: `${movesUrl}?move-date=${date}`,
```

Comment:
There is `res.locals.MOVES_URL` which exists on the locals to take the user back to the dashboard view they started on so this could be used in the view to avoid needing to define in the controller.